### ISSUE TITLE
No headphone sound on Asus C302 (Cave)

### ISSUE THREAD

**Issue by lbilli** [WeirdTreeThing/chromebook-linux-audio#133]
After running the script on an Asus C302CA (Cave) with openSUSE Tumbleweed, I get:
- builtin speakers work great
- builtin mic works, though low volume
- headphone mic works great
- headphone speakers produce no sound, just a single click when trying to play anything

Also, when inserting the jack, the headphone mic gets automatically selected, whereas the headphone speakers are not.

The only log message that looks it might be related:
`kernel: nau8825 i2c-10508825:00: No 'mclk' clock found, assume MCLK is managed externally`

Any suggestions?

**Comment by lbilli** [comment]
As I saw in a recent commit an update to the avs topology for `nau8825`, I decided to give it a try and indeed it mostly fixed my issue.
Headphones sound is now working, the only minor issue being a fairly audible click when sound starts.

So, I'm wondering what's the difference with the `nau8825` topology shipped in the 2024.02 release of `thesofproject/avs-topology-xml`.

Finally, while I was fiddling with the ucm conf files for `avs_nau8825`, I was wondering if, in the `Mic` section, `JackControl` should point to `Headset Mic Jack` instead of `Headphone Jack`.
